app.py

```diff
@@ -46,9 +46,6 @@
 
 _LOGGER = logging.getLogger("thoth.graph_refresh_job")
 
-_SOLVER_OUTPUT = os.getenv(
-    "THOTH_SOLVER_OUTPUT", "http://result-api/api/v1/solver-result"
-)
 _PACKAGE_ANALYZER_OUTPUT = os.getenv(
     "THOTH_PACKAGE_ANALYZER_OUTPUT", "http://result-api/api/v1/package-analysis-result"
```

Comment:
what about this @fridex ?

Reply:
I'll drop package analyzer in a separate PR. We should substitute it with a workflow obtaining different aspects of packages.

Reply:
awesome, thanks